### PR TITLE
add log retrieval command

### PR DESCRIPTION
**Pull request by Oscaz**
solves #11

### CODE REVIEW

**Review by ollibowers — COMMENTED** [review]
Looks good!

File: extensions/log.py [hunk 1]
```diff
@@ -80,6 +82,26 @@ def load_directory(self):
...
 
+    @commands.command()
+    @commands.has_permissions(administrator=True)
+    async def logs(self, ctx, type='Invalid', amount='100'):
+        if type != 'general' and type != 'error' and type != 'command_error' or (amount != 'all' and not amount.isdigit()):
+            await ctx.send('Usage: `$logs (general | error | command_error) [amount=100 | all]`')
+            return
+        with open(f'{self.root_dir}{type}.log', 'r') as file:
+            if amount == 'all':
+                await asyncio.sleep(0.1) # This is required, as this command executes something is
```
Comment (insou22) [inline extensions/log.py:93]: uh

File: extensions/log.py [hunk 2]
```diff
@@ -80,6 +82,26 @@ def load_directory(self):
...
+                                            # written to the log file, which is temporarily cleared or
```
Comment (insou22) [inline extensions/log.py:94]: uh

Comment (XenocidePC) [reply extensions/log.py:94]: I've seen worse.

Comment (Wal-eed) [reply extensions/log.py:94]: We'll be moving to Discord.js, so temporarily this will suffice.

**Pull request merged**